### ISSUE TITLE
theme for website

### ISSUE THREAD

**Issue by fnriv** [cse110-sp23-group3/8-ball#3]
- [ ] traditional 8ball
- [ ] brand theme (lions)
- [ ] powell the fortune teller

**Comment by phuanh004** [comment]
Could be just a basic theme with main/ secondary color

**Comment by fnriv** [comment]
I agree; i know chay made some branding with the lion so maybe we can follow that color scheme and maybe add our brand somewhere

**Comment by phuanh004** [comment]
Sounds good. Wait for everyone's input!

**Comment by nsyousef** [comment]
I would be good with branding the app with our lion theme. If you want, since we are called "Powell's Pride Tech" we could perhaps work Professor Powell in there too somehow.

**Comment by dolivares11** [comment]
I like the idea of using our brand theme, but powell the fortune teller could provide some comedic effect. I tried using voice cloning AI to try to get Powell's voice, but I had no success.
